tasks/32_numbers-to-reversed-array/numbers-to-reversed-array.js

```diff
@@ -0,0 +1,6 @@
+function numberToReversedArray(number) {
+  return number.toString().split('').reverse().map((x)=> Number(x));
```

Comment:
You can pass `Number` function straight to `.map()` method, it will pass value anyway.